test/e2e/virtual/apiexport/virtualworkspace_test.go

```diff
@@ -326,7 +326,23 @@ func TestAPIExportPermissionClaims(t *testing.T) {
 	bindConsumerToProviderWithPermissionClaims(ctx, consumerWorkspace, serviceProviderWorkspace, t, kcpClusterClient, cfg, identityHash)
 
 	framework.Eventually(t, func() (success bool, reason string) {
-		// Get the binding and make sure that observed permission claims are all set.
+		// Get the binding and make sure that observed export permission claims are all set.
+		binding, err := kcpClusterClient.ApisV1alpha1().APIBindings().Get(logicalcluster.WithCluster(ctx, consumerWorkspace), "cowboys", metav1.GetOptions{})
+		if err != nil {
+			return false, err.Error()
+		}
+
+		for _, claim := range binding.Status.ExportPermissionClaims {
```

Comment:
Would there ever be a case where `Status.ExportPermissionClaims` and `Status.AppliedPermissionClaims` would not be the same?

Reply:
Potentially, yes. It can take time to label things (this has to finish before AppliedPermissionClaims is updated)